haskell/src/SwiftNav/SBP/Acquisition.hs

```diff
@@ -0,0 +1,26 @@
+module SwiftNav.SBP.Acquisition where
+
+import Data.Int
+import Data.Text
+import Data.Word
+
+
```

Comment:
Oh yeah, I think we'll want instances of `Binary` and its `put` and `get` functions.